packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -219,10 +247,13 @@ class ExpiringMultiPartyClient {
         sponsor: this.activeSponsors[index],
         withdrawalRequestPassTimestamp: position.withdrawalRequestPassTimestamp,
         withdrawalRequestAmount: position.withdrawalRequestAmount.toString(),
-        numTokens: position.tokensOutstanding.toString(),
-        amountCollateral: this.toBN(position.rawCollateral.toString())
+        numTokens: this.toBN(position.tokensOutstanding.toString()) // Apply the current funding rate to the sponsor debt.
```

Comment:
this is the only change needed for the client: the `numTokens` (sponsor outstanding token debt) is scaled by the current `CumulativeFundingRateMultiplier`